### ISSUE TITLE
Will we ever be able to use `pandoc-citeproc` with RMarkdown again?

### ISSUE THREAD

**Issue by briandk** [rstudio/rmarkdown#583]
First, I love love love RMarkdown. Thank you for making it.

Second, I've been writing all my manuscripts using [pandoc-style citations][1] as specified on [RStudio's markdown page](http://rmarkdown.rstudio.com). For example:

```
Blah blah [see @doe99, pp. 33-35; also @smith04, ch. 1].

Blah blah [@doe99, pp. 33-35, 38-39 and *passim*].

Blah blah [@smith04; @doe99].
```

**But**, as of https://github.com/rstudio/rmarkdown/commit/b6e3f39f731b28af5ff1823c0de8714d8fb78df7 RMarkdown [no longer supports][2] `pandoc-citeproc`:

> ```
> * For PDF/LaTeX output, citations are processed via natbib or biblatex instead
>   of pandoc-citeproc. The ciation package natbib or biblatex can be chosen using
>   the argument `citation_package` in `pdf_document()`, `beamer_presentation()`,
>   and `tufte_handout()`. LaTeX is compiled to PDF via latexmk
>   (https://www.ctan.org/pkg/latexmk/); if it is not installed, a simple
>   emulation will be used (run pdflatex/xelatex/lualatex, bibtex, and makeindex
>   a few times). You are recommended to install latexmk, and please note latexmk
>   requires a Perl installation (this is important especially for Windows users).
> ```

So,

1. Will `pandoc-citeproc` support ever be coming back?
2. Is there any way I can use `pandoc-citeproc` given how Rmarkdown works now?

Basically, I'm trying to figure out what I need to do with all my pandoc-style-citation-formatted RMarkdown manuscripts because they don't render properly anymore :-(

[1]: http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation-syntax 
[2]: https://github.com/rstudio/rmarkdown/blob/b6e3f39f731b28af5ff1823c0de8714d8fb78df7/inst/NEWS#L11-L18

**Comment by yihui** [comment]
This issue has been fixed in the latest version of rmarkdown (0.9.1): https://github.com/rstudio/rmarkdown/blob/master/inst/NEWS The source package is on CRAN now, and you may install it with

```r
install.packages('rmarkdown', repos = 'https://cran.rstudio.com', type = 'source')
```

pandoc-citeproc will always be used unless you want to use natbib or biblatex to process the citations.

**Comment by briandk** [comment]
@yihui - I JUST saw that! Sorry to bother you :-(

But, just to be clear, with the new (0.9.1) behavior, I can specify a YAML block like this, and `rmarkdown` will just pass the arguments along to `pandoc` and `pandoc-citeproc`, as it used to (around version 0.8.0 and before):

```yaml
title: "My publishable manuscript"
author: "[Blinded for Review]"
documentclass: "article"
output:
  pdf_document:
    citation_package: 'none'
    pandoc_args: [
      --bibliography=bibliography.bibtex,
      --csl=apa.csl,
      --filter=pandoc-citeproc
    ]
```

Follow up: Should I be explicit with `citation_package: 'none'` in my YAML?

**Comment by yihui** [comment]
You don't need to specify `citation_package`. Actually there is a much cleaner way to specify bibliography and csl:

```yaml
title: "My publishable manuscript"
author: "[Blinded for Review]"
documentclass: "article"
output:
  pdf_document: default
bibliography: bibliography.bibtex
csl: apa.csl
```

When rmarkdown detects `bibliography` in YAML, it will call `pandoc-citeproc` automatically.

**Comment by briandk** [comment]
@yihui - Whoaaaa. Thanks!
